group/static/group/js/editGroup.js

```diff
@@ -0,0 +1,74 @@
+/*
+The MIT License (MIT)
+Copyright (c) 2014 NTHUOJ team
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+$(document).ready(function() {
+    add_attribute('id_coowner', 'class', 'searchable');
+    add_attribute('id_trace_contest', 'class', 'searchable')
+    enable_search();
+    widen_multiselect();
+    $('input.search-input').quicksearch();
+    $('.help-block').hide()
+});
+
+function enable_search() {
+    $('.searchable').multiSelect({
+        selectableHeader: 
+        "<input type='text' class='search-input form-control' autocomplete='off' placeholder='Search/Add...'>",
+        selectionHeader: 
+        "<input type='text' class='search-input form-control' autocomplete='off' placeholder='Search/Delete...'>",
+        selectableFooter: "<div class='all-header' style='text-align:center;'>All</div>",
+        selectionFooter: "<div class='selected-header' style='text-align:center;'>Now Selected</div>",
```

Comment:
Just a suggestion. Is there any possible change to avoid inject html directly?
Just append an element and update the content of the element.